index.js

```diff
@@ -60,13 +60,69 @@ animalToRemove.remove()
 
 /***** Deliverable 1 *****/
 function toggleColor(element) {
-    if (element.style.color === "green") {
+    if (element.style.color === "red") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
+    console.log(element)
 }
 
+
+// header.addEventListener('click', function(event){
+//     toggleColor(event.target)
+// })
+// IF header has children, use header > event.target
+
+header.addEventListener('click', function(){
+    toggleColor(header)
+})
+// console.log('click')
+
```

Comment:
Good! 

- When you learn about arrow functions (ES6 feature), you could also write this solution like shown below. This is the same exact functionality!

```
header.addEventListener("click", () => toggleColor(header))
```